src/www/ui/template/ui-report-conf.html.twig

```diff
@@ -121,26 +121,29 @@
           </td>
           <td align="left">
             <label><input type="checkbox" name="noDependency" {{ noDependency }} />{{ "no dependencies found, neither in source code nor in binaries"|trans }}</label><br />
-            <label><input type="checkbox" name="dependencySource" {{ dependencySource }} />{{ "dependencies found in source code (see obligations)"|trans }}</label><br />
-            <label><input type="checkbox" name="dependencyBinary" {{ dependencyBinary }} />{{ "dependencies found in binaries (see obligations)"|trans }}</label>
+            <label><input type="checkbox" onclick="toggleTextArea1();" id="dependencySource" name="dependencySource" {{ dependencySource }} />{{ "dependencies found in source code (see obligations)"|trans }}</label><br />
+            <label><input type="checkbox" onclick="toggleTextArea1();" id="dependencyBinary" name="dependencyBinary" {{ dependencyBinary }} />{{ "dependencies found in binaries (see obligations)"|trans }}</label>
+            <label><textarea {{ styleTextArea1 }} id="dependencyBinarySource" name="dependencyBinarySource">{{ dependencyBinarySource|e }}</textarea></label>
           </td>
         </tr>
         <tr>
           <td align="left">
             {{ "Export restrictions by copyright owner"|trans }}
           </td>
           <td align="left">
-            <label><input type="checkbox" name="noExportRestriction" {{ noExportRestriction }} />{{ "no export restrictions found"|trans }}</label><br />
-            <label><input type="checkbox" name="exportRestriction" {{ exportRestriction }} />{{ "export restrictions found (see obligations)"|trans }}</label>
+            <label><input type="checkbox" onclick="toggleTextArea2();" id="noExportRestriction" name="noExportRestriction" {{ noExportRestriction }} />{{ "no export restrictions found"|trans }}</label><br />
+            <label><input type="checkbox" onclick="toggleTextArea2();" id="exportRestriction" name="exportRestriction" {{ exportRestriction }} />{{ "export restrictions found (see obligations)"|trans }}</label>
+            <label><textarea {{ styleTextArea2 }} id="exportRestrictionText" name="exportRestrictionText">{{ exportRestrictionText|e }}</textarea></label>
           </td>
         </tr>
         <tr>
           <td align="left">
             {{ "Restrictions for use by copyright owner \n<br/> (e.g. not for Nuclear Power)"|trans }}
           </td>
           <td align="left">
-            <label><input type="checkbox" name="noRestriction" {{ noRestriction }} />{{ "no restrictions for use found"|trans }}</label><br />
-            <label><input type="checkbox" name="restrictionForUse" {{ restrictionForUse }} />{{ "restrictions for use found (see obligations)"|trans }}</label>
+            <label><input type="checkbox" onclick="toggleTextArea3();" id="noRestriction" name="noRestriction" {{ noRestriction }} />{{ "no restrictions for use found"|trans }}</label><br />
```

Comment:
Please update the function name from `toggleTextArea3` to something more meaningful.